CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 0.2.4
+* add setting `terminalColors`
+
```

Comment:
Though it probably should be a SEMVER-minor, but I haven't been really rigorous on that. :/